peregrine/iqgen/bits/encoder_1bit.py

```diff
@@ -0,0 +1,82 @@
+# Copyright (C) 2016 Swift Navigation Inc.
+#
+# This source is subject to the license found in the file 'LICENSE' which must
+# be be distributed together with this source. All other rights reserved.
+#
+# THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+# EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+
+import numpy
+
+"""
+The :mod:`peregrine.iqgen.encoder_1bit` module contains classes and functions
+related to generating single bit signal output.
+
+"""
+
+import peregrine.iqgen.if_iface
+from peregrine.iqgen.bits.encoder_base import Encoder
+
+GPS_L1_INDEX = peregrine.iqgen.if_iface.Chip.GPS.L1.INDEX
+GPS_L2_INDEX = peregrine.iqgen.if_iface.Chip.GPS.L2.INDEX
```

Comment:
These aren't used in this file - are they meant to be imported by other modules?